modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/information/ElasticsearchSearchEngineInformation.java

```diff
@@ -261,6 +271,12 @@ private void _addConnectionInformation(
 		}
```

Comment:
What are these labels? If they are related to the connection names we should log them.